CONTRIBUTING.md

```diff
@@ -60,7 +60,7 @@ Try to avoid introducing new code analysis warnings. Currently the codebase is f
 
 ## Resharper Artifacts
 
-Please do not add Resharper suppressions to code using comments. You may tweak your local Resharper settings but do not commit these to the repo. A definitive Resharper settings file is in the pipeline.
+Please do not add Resharper suppressions to code using comments. You may tweak your local Resharper settings but do not commit these to the repo.

```

Comment:
It might be worth explaining how to achieve this. When configuring e.g. inspection severity, you can choose where to save the settings:
- Solution "FakeItEasy" personal
- Solution "FakeItEasy" team-shared
- This computer

You should pick either the "personal" or "this computer" option.